craco.config.js

```diff
@@ -1,3 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: CC0-1.0
```

Comment:
Why is this CC0 instead of AGPL3.0-only?

Reply:
I put all config files as CC0

Reply:
Why

Reply:
This was suggested by https://reuse.software/tutorial/#insignificant-files

Reply:
For files like config.json, .renovaterc.json etc that indeed makes sense, but I'd see this more as a source code file than a config file.

Reply:
okay anything else I should put under AGPLv3

Reply:
this and package.json changed